### ISSUE TITLE
Undefined symbols for architecture x86_64 after updating cocoapods to 0.38.2

### ISSUE THREAD

**Issue by myexec** [CocoaPods/CocoaPods#4224]
Hi guys,
I have a problem after updating cocoapods to the latest version 0.38.2. I install a pod library to a static library and then include this static library in the main app. After upgrading cocoapods and running 'pod install', the main app won't build with the following error:

Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_CPTTheme", referenced from:
      objc-class-ref in libStaticLib.a(StaticLib.o)

I have uploaded a sample project here:
https://github.com/myexec/Pods-Problem

Any help will be appreciated.
Thanks.

**Comment by samdmarshall** [comment]
I checked the coreplot issue tracker and it looks like the recommended fix for this is to use 1.6 or newer: https://github.com/core-plot/core-plot/issues/174 (it looks like you are using 1.3).

I suspect that updating to 0.38.2 uncovered this and that you weren't building for 64bit simulator before.

**Comment by myexec** [comment]
I actually figured it out by myself. It looks like for some reason CorePlot doesn't get linked to the main app. This can be fixed by adding -l CorePlot flag to the Other Linker Flags or what I did was to include the CorePlot for the both projects in Podfile:

platform :ios, '7.0'
workspace 'test9'

target :StaticLib do
  xcodeproj 'test9/StaticLib/StaticLib.xcodeproj'
  pod 'CorePlot','1.3'
end

target :test9 do
  xcodeproj 'test9.xcodeproj'
  pod 'CorePlot','1.3'
end


This fixed the problem.
Thanks for your help samdmarshall. I appreciate it.
